packages/cli/src/commands/generate/forms/generate_forms_command.ts

```diff
@@ -84,6 +85,11 @@ export class GenerateFormsCommand
       uiOutDir: outDir,
       apiUrl,
       modelsFilter: args.models,
+      logCallback: (filePath: string) => {
```

Comment:
If we ever want to log something else within these commands, this isn't a very extensible abstraction. Perhaps we should just pass the printer into the generate_forms_command and generate_config_command and they can handle the format of exactly what gets logged

Reply:
Thats another solution I considered as well. To make Typescript compiler happy, I will have to add core-platform dependency in `client-config`, `form-generator`, `model-generator`.

Reply:
We should file a tech-debt item to make Printer a type in platform-core and have the Printer in cli-core implement that type.

For now though I think we could just change this input to
```ts
log: printer.log
```
so that we aren't making assumptions about what is being logged

Reply:
Updated types and interface with TODO comment to update them in the future.